people/michaela.json

```
@@ -0,0 +1,12 @@
+{
+ "key": "michaela",
+ "name": "Ing. Michaela Plisková",
+ "nick": "Michaela",
+ "email": "michaela.pliskova@mild.blue",
+ "title_cs": "Operations manažerka",
+ "title_en": "Operations manager",
+ "title_de": "Operations managerin",
```

```suggestion
 "title_de": "Operations Managerin",
```

Maličká úprava pre jednotnosť.